.travis.yml

```diff
@@ -7,7 +7,17 @@ r_packages:
 
 after_success:
   - Rscript -e 'library(covr); codecov()'
-  
+  - Rscript -e 'pkgdown::build_site()'
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GITHUB_PAT
```

Comment:
Noting that @thevaachandereng set up the `$GITHUB_PAT` environment variable at Travis with his personal access token 
![image](https://user-images.githubusercontent.com/3532898/42939976-5b47a476-8b1d-11e8-8b18-8f061aa43e7b.png)

These Travis docs explain more and provide other configuration options https://docs.travis-ci.com/user/deployment/pages/